### DISCUSSION TITLE
Single meeting duplicates in meeting guide app

### DISCUSSION THREAD

**Discussion by Historyn** [code4recovery/12-step-meeting-list#1467] [Q&A] [answered]
Our URL:   aa-cedarrapids.org
I have entered the information for a meeting called Marion Midweek AA and it comes up fine in the meeting guide both on the website and the app.  There is another entry for a similar group, however, called Marion Mid-Week Group that appears in the meeting guide app, but does NOT appear in the database or on the website.  How do I find and eliminate this non-existent Marion Mid-Week Group?
Thanks!

**Comment by joshreisner** [comment, marked as answer]
it's very common for multiple overlapping service entities to list the same meeting. meeting guide filters out duplicates in these situations by using addresses. 

in this case the you are representing district 8: https://www.aa-cedarrapids.org/meetings/marion-midweek-aa/

also the area is listing this meeting twice. once at the same address as you, this is being blocked because it uses the same address as you: https://www.aa-iowa.org/meetings/marion-midweek-aa/

also they are listing this meeting at `13th St & 7th Ave`: https://www.aa-iowa.org/meetings/marion-mid-week-group/

i would ask the area to drop this listing and the problem should resolve itself

**Reply by joshreisner** [reply]
what has been making this a challenge though is that google is a little confused about where `1298 7th Ave` is - you can see it comes back with two options here:

<img width="1840" alt="Screenshot 2024-07-14 at 7 49 01 PM" src="https://github.com/user-attachments/assets/19efff69-dd75-4ec7-bbed-afa6941d4126">

the area may be listing it at `13th St & 7th Ave` because that improves the map location. in this case they might want to override the address like this: https://wordpress.org/plugins/12-step-meeting-list/#i%20need%20to%20correct%20a%20meeting%20address%20or%20change%20a%20pin%27s%20location

**Comment by Historyn** [comment]
Thanks for your help, Josh,
I think we finally have it sorted -- you were corrected that the Area webmaster had manually added the entry and consequently it wasn't deleted when they refreshed their page after our changes.
Cheers,Kathy 
    On Sunday, July 14, 2024 at 09:46:58 PM CDT, Josh Reisner ***@***.***> wrote:  
 
 


it's very common for multiple overlapping service entities to list the same meeting. meeting guide filters out duplicates in these situations by using addresses.

in this case the you are representing district 8: https://www.aa-cedarrapids.org/meetings/marion-midweek-aa/

also the area is listing this meeting twice. once at the same address as you, this is being blocked because it uses the same address as you: https://www.aa-iowa.org/meetings/marion-midweek-aa/

also they are listing this meeting at 13th St & 7th Ave: https://www.aa-iowa.org/meetings/marion-mid-week-group/

i would ask the area to drop this listing and the problem should resolve itself
